tools/abel/README.md

```diff
@@ -1,191 +1,233 @@
+```
+
+
+### Obtain a GCP OAuth Client Credential
+
+Contact your IoT Technical Program Manager and ask for an OAuth client credential.json file for authenticating against Google Sheets API. Use the credential.json file for ABEL's --credential command line argument discussed in the [ABEL Arguments](#abel-arguments) section below.
+
+
+## Using ABEL
+
+When using ABEL there is a set of commands that should be run each time to ensure that the tool and all its requirements are up to date within your local repo. These instructions can be found in the [General Use](#general-use) section. The actual command to run ABEL will use a different set of arguments depending on the action being performed. Please go to the section for the action you are trying to perform to see instructions on which [arguments](#abel-arguments) to pass in.
 
-  ```
-  git clone https://github.com/google/digitalbuildings.git
-  ```
 
-2. Navigate to the [DigitalBuildings tooling library](../../tools/)
+### General Use
 
+This section describes commands that should be run every time ABEL is used to make sure the program and validation are up to date, as well as an overview of all of the arguments possible to pass into ABEL.
+
+
+#### Update your Repository Clone and Activate your Virtual Environment
+
+The following commands will make sure the Digital Buildings repository as a whole is up to date, that your virtual environment is active, and that you have the most up to date packages installed for ABEL:
+
+1. In the command line, navigate to the Digital Buildings repository cloned on your computer. If the Digital Buildings repository is not in your home directory, replace “~” with the folder it exists within.
   ```
-  cd digitalbuildings/tools/
+  cd ~/digitalbuildings/
   ```
 
-3. Create a virtual environment with an environment name, in this example: `tooling`
+2. Pull any updates made to the repository since your last pull.
+  ```
+  git pull
+  ```
 
+3. Navigate to your tooling virtual environment. Replace “tooling” with the name you gave your venv. 
   ```
-  python3 -m venv tooling
+  cd /tools/
   ```
 
-4. Activate your virtual environment
+4. Activate your virtual environment.
 
-  * MacOs/Linux:
-  
-    ```
-    source tooling/bin/activate
-    ```
-  
-  * Windows:
+    * MacOs/Linux:
+      ```
+      source tooling/bin/activate
+      ```
   
-    ```
-    tooling\Scripts\activate
-    ```
-
-5. Depending on your operating system, run either of the global setup scripts to configure dependencies
+   * Windows:
+      ```
+      tooling\Scripts\activate
+      ```
 
-  * Linux/MacOs:
+5. Configure dependencies/packages.
 
-    ```
-    ./pip_install.sh
-    ```
-  
-  * Windows:
+    * MacOs/Linux:
+      ```
+      ./pip_install.sh
+      ```
   
-    ```
-    pip_install.bat
-    ```
-
-6. Navigate to the [ABEL directory](./)
+    * Windows:
+      ```
+      pip_install.bat
+      ```
 
+6. Navigate to the ABEL directory.
 ```
-cd abel
+cd /abel/
```

Comment:
forward slashes are unnecessary for Linux (and maybe MacOs, too?) -- suggest splitting commands into Windows OS and MaxOs/Linux